roles/lib_utils/src/class/repoquery.py

```diff
@@ -48,7 +48,7 @@ def process_versions(query_output):
 
         version_dict = defaultdict(dict)
 
-        for version in query_output.split('\n'):
+        for version in query_output.decode().split('\n'):
```

Comment:
This will be hit by other items using ``_run`` ([local ref](https://github.com/jpeeler/openshift-ansible/blob/7fb814d6197c581b0dfcc974a5b0a4646eab6cea/roles/lib_utils/library/repoquery.py#L358)). It may be a better fix to ``decode()`` before ``_run`` returns.

Here is what was done in [lib_openshift](https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/src/lib/base.py#L259).